test/powershell/Language/Scripting/ParameterBinding.Tests.ps1

```diff
@@ -1,7 +1,4 @@
-﻿Import-Module $PSScriptRoot\..\..\Common\Test.Helpers.psm1 -ErrorAction SilentlyContinue
```

Comment:
Removing the module imports just doesn't seem right to me.  If you invoke-pester directly, it doesn't even tell you what is wrong.